system/Autoloader/Autoloader.php

```diff
@@ -169,7 +169,19 @@ public function register()
 	 */
 	public function addNamespace($namespace, $path)
 	{
-		$this->prefixes[$namespace] = $path;
+		if (isset($this->prefixes[$namespace]))
+		{
+			if (is_string($this->prefixes[$namespace]))
+			{
+				$this->prefixes[$namespace] = [$this->prefixes[$namespace]];
+			}
+			$this->prefixes[$namespace] = array_merge($this->prefixes[$namespace], [$path]);
```

Comment:
Please leave a blank line between logical blocks and the next line. It makes for more readable code. Think of them like paragraphs in prose. :)